templates/nas-grp-base.html.twig

```diff
@@ -159,8 +159,8 @@
         });
     </script>
 
-    <script>
-        window.PELAGOS_TEMPLATE_PROPS = {"hideSearchTextBlock": true, "fundingCycleFacet": true};
+    <script type="text/javascript">
+        window.PELAGOS_TEMPLATE_PROPS = {"BaseTemplateName": "GRP", "hideSearchTextBlock": true, "fundingCycleFacet": true};
```

Comment:
I think we should get rid other variables for individual rendering, and just have BaseTemplateName and have a util js file which gets griidc text or grp text

Reply:
Each "app" could import some type of language/verbiage file. Or we could use different templates with properties in it.
It's only a few now, but if there will be a lot more we should change it.

Reply:
this is only a start

Reply:
I've created a dummy PR, where I've committed the code of what might be a possible way to do this
https://github.com/griidc/pelagos/pull/717